src/styles/sass/main.scss

```diff
@@ -6,87 +6,340 @@
 @import "type";
 
 // Navigation Control
-
-.controls {
-  color: $nypl-white;
+// Temp grayscaling for ease of styling...
+body {
+  // filter: grayscale(1);
+  // filter: grayscale(0);
   font-family: $sans-serif-stack;
   font-size: $basefont;
-  height: 4.65rem;
+  margin: 0 auto !important;
```

Comment:
if you're overriding https://github.com/NYPL-Simplified/webpub-viewer/blob/master/examples/alice/index.html#L9 just change that line